CHANGELOG.rst

```diff
@@ -4,6 +4,87 @@
 Synapse Changelog
 *****************
 
+v2.143.0 - TBD
+==============
+
+Model Changes
+-------------
+- Update to the ``crypto`` model.
+  (`#3256 <https://github.com/vertexproject/synapse/pull/3256>`_)
+
+  Updated Types
+  -------------
+
+  ``hex``
+    The ``zeropad`` option has been changed from a ``bool`` to an ``int``.
+    It now indicates how many bytes to left pad a value with zeros (``0x00``).
```

Comment:
This kinds of makes it sound like we prepend `zeropad` number of bytes instead of padding out to `zeropad` size.

```suggestion
    The ``zeropad`` option has been changed from a ``bool`` to an ``int``.
    It now indicates the size to which a value should be zero (``0x00``) padded.
```

Reply:
`It may now be used to specify the zero extended length of the hex string.`